### ISSUE TITLE
No issue But have Questions.

### ISSUE THREAD

**Issue by josephchrzempiec** [MerlotMachina/ESP8266BitcoinMiner#1]
Hello i was wondering if it is really possible to mine from a ESP8266. And it looks like you are doing it. Is it really possible?

**Comment by huseman21** [comment]
Yes its possible but it would take like a billion esp's and millions $$ in electric bills a day to have any chance of a single return.

**Comment by armeni0n** [comment]
how to use this
